src/expr.rs

```diff
@@ -37,6 +37,7 @@ pub enum SimpleExpr {
     AsEnum(DynIden, Box<SimpleExpr>),
     Case(Box<CaseStatement>),
     Constant(Value),
+    Check(String),
```

Comment:
Hey @LemarAb, thanks for contributing!!

We should represent the check expression with `SimpleExpr`.